SoftwareGuide/Latex/ITKSoftwareGuide-Book1.tex

```diff
@@ -147,14 +147,15 @@ \part{Development Guidelines}
 \input{DevelopmentGuidelines/SoftwareProcess.tex}
 
 
-\begin{appendices}
+\renewcommand{\thepart}{}
+\renewcommand{\partname}{}
+\part{Appendices}
+\appendix
```

Comment:
What happens if `\begin{appendices}` and `\end{appendices}` are kept? I've read that they are better for long documents vs. using `\appendix`.